influunt-app/app/plugins/roundSlider-1.3/roundslider.js

```diff
@@ -211,7 +211,15 @@
 
             this._bind(this.input, "blur", this._focusOut);
             this._bind(this.input, "change", this._focusOut);
-        },
+
+            this.input.keypress(function(event) {
+              var charCode = (typeof event.which == "undefined") ? event.keyCode : event.which;
```

Comment:
Essa comparação com `undefined` só vai retornar `true` quando o valor de `event.which` for a string "undefined". Para verificar se é `undefined` mesmo, pode comparar com o valor direto (`event.which === undefined`), ou então verificar o tipo da variável: `typeof event.which === 'undefined'`

![screen shot 2017-09-04 at 14 26 23](https://user-images.githubusercontent.com/194442/30035121-126a09d2-917d-11e7-8ace-5af49dfc25e9.png)